azure-pipelines.yml

```diff
@@ -56,3 +58,38 @@ jobs:
         tox -e py37
         tox -e py36
         codecov
+
+- job: Linux_Build
+  pool: {vmImage: 'Ubuntu-16.04'}
+  steps: 
+    - task: UsePythonVersion@0
+    - bash: |
+        python -m pip install --upgrade pip
+        python -m pip install tox pytest click cython numpy pandas plotly scitrack tqdm tinydb cibuildwheel==0.11.1 pytest-azurepipelines
```

Comment:
Can we change these massive long lines to `python -m pip install .[dev] cibuildwheel==0.11.1 pytest-azurepipelines`?

That should trigger install of the dev dependencies (and thus it's an implicit test we haven't forgotten something) while also adding the other just for CI dependencies.

Reply:
I see that `pytest-azurepipelines` is already in `[dev]`, so I will omit that, but the change has been made, will push shortly.